### ISSUE TITLE
Title here

### ISSUE THREAD

**Issue by MEgapixrls** [tukui-org/ElvUI#1065]
### What is the issue you are having?

Dear ElvUI developers,

I am writing to report a bug that I have encountered since the latest update of ElvUI.

After the latest update, the text color inside the Player Frame has been automatically changed to a color that is slightly less visible.

I believe this is a bug, and I would appreciate it if you could help me find a solution.

Thank you for your time and attention.

![123123](https://github.com/tukui-org/ElvUI/assets/147964438/5f24409a-e1cc-4c99-9d63-3fe3f6e998fe)

### What is the expected behavior?

.

### What actually happened?

.

### Suggested Solution/Workaround

.

### Errors

```shell
.
```

### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/development/CHANGELOG.md).
- [X] I have verified ElvUI is up to date by checking /estatus.
- [X] I have verified my issue doesn't exist in the ElvUI Issue Tracker.
- [X] I have not read any of these steps, please close my issue when you see it.
- [X] I have verified my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the three called 'ElvUI', 'ElvUI Libraries', and 'ElvUI Options' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

**Comment by MEgapixrls** [comment]
![image](https://github.com/tukui-org/ElvUI/assets/147964438/07aa699d-a373-46be-8ebd-32a7922acd39)

**Comment by Repooc** [comment]
![image](https://github.com/tukui-org/ElvUI/assets/1479745/64111b4b-146c-4f2f-931c-b38743251761)
not a bug... per changelog as you can see in the picture attached, referring to New Font Outline setting
tldr; change font outline to shadow outline for the previous look

**Comment by MEgapixrls** [comment]
It was my mistake. I am a Korean and I just clicked on the latest update without reading the changelog. I hope you are always healthy and have a good day.

**Comment by syswis** [comment]
Does not work with "Custom Text" in unit frames, no matter the settings i chose it's always without outline/shadow.
